### PR TITLE
*: make tests more stable

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
Currently, the store limit related tests are not very stable.

### What is changed and how it works?
This PR changes some test configurations to make them more stable.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: server/coordinator_test.go [hunk 1]
```diff
@@ -795,32 +795,37 @@ func (s *testOperatorControllerSuite) TestStoreOverloaded(c *C) {
...
-	time.Sleep(2 * time.Second)
+
+	oc.SetAllStoresLimit(10)
+	for i := 0; i < 10; i++ {
+		op1 = lb.Schedule(tc)[0]
+		c.Assert(op1, NotNil)
+		c.Assert(oc.AddOperator(op1), IsTrue)
+		c.Assert(oc.RemoveOperator(op1), IsTrue)
+	}
+	time.Sleep(1 * time.Second)
```
Comment (shafreeck) [inline server/coordinator_test.go]: It is hard for me to understand all the cases in this test. Would you like to add more comments for each important case?

Ex. Why sleep 1s? Why we can schedule 100 times with the store limit 10 per minute?

File: server/coordinator_test.go [hunk 2]
```diff
@@ -830,7 +835,7 @@ func (s *testOperatorControllerSuite) TestStoreOverloadedWithReplace(c *C) {
 	cfg, opt, err := newTestScheduleConfig()
 	c.Assert(cfg, NotNil)
 	c.Assert(err, IsNil)
-	cfg.StoreBalanceRate = 10
+	cfg.StoreBalanceRate = 30
```
Comment (shafreeck) [inline server/coordinator_test.go]: Could you explain why 30 is the proper value here?

Comment (shafreeck) [reply server/coordinator_test.go]: I mean, add the comment.

**Comment by disksing** [comment]
PTAL @lhy1024

File: server/coordinator_test.go (hunk 2 above)
Comment (rleungx) [reply server/coordinator_test.go]: Done. PTAL @shafreeck

File: server/coordinator_test.go [hunk 3]
```diff
@@ -795,32 +795,42 @@ func (s *testOperatorControllerSuite) TestStoreOverloaded(c *C) {
...
+	c.Assert(tc.addRegionStore(2, 100), IsNil)
 	c.Assert(tc.addRegionStore(1, 10), IsNil)
 	c.Assert(tc.addLeaderRegion(1, 2, 3, 4), IsNil)
-	start := time.Now()
+	region := tc.GetRegion(1).Clone(core.SetApproximateSize(60))
+	tc.putRegion(region)
 	op1 := lb.Schedule(tc)[0]
 	c.Assert(op1, NotNil)
 	c.Assert(oc.AddOperator(op1), IsTrue)
+	c.Assert(oc.RemoveOperator(op1), IsTrue)
```
Comment (lhy1024) [inline server/coordinator_test.go]: why do we need to remove operator

Comment (rleungx) [reply server/coordinator_test.go]: It will occupy the position with the same region ID.

**Comment by rleungx** [comment]
Yes. Maybe we can do it in another PR?

**Pull request merged**